GUI/run.js

```diff
@@ -341,23 +342,17 @@ async function main(commandLine) {
 
     console.log("OoT Randomizer GUI is booting up");
 
-    //Verify basic environment
-    if (!fs.existsSync("./node_modules"))
-        await setupNodeEnvironment(true);
-
-    //Verify binary folder exists
-    if (!fs.existsSync("./node_modules/.bin")) {
```

Comment:
Do we no longer have the .bin folder? Am wondering if we should still check for it.

Reply:
The .bin directory still exists. The check didn't do anything since a commit in 2019, so I figured you might have had a reason to remove it. If there was no reason we can add it to the others to perform a clean install of all packages.

@dragonbane0 Do you remember if this was on purpose? 
https://github.com/TestRunnerSRL/OoT-Randomizer/commit/10e5fffed8ebc4313e887e407f548d9f30535d21#diff-9daf8496654d685688dbea6e1c8f1a69276878a773a105e14627fb6f574121a0R312

Reply:
@BenjaminBrandmeier I honestly don't remember, but it looks accidental to me. At least if the .bin folder still exists, I see no reason not checking for it. Continuing on without it present will never work, so I don't see why I commented out the throw. Maybe it was a quick test

Reply:
Thanks for the quick response! I have added it to the list, triggering a fresh install via npm ci.